public/class-award-force-api-v2.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use GuzzleHttp\Client;
+
+class AwardForceAPIV2 {
+
+    private $apiUrl = 'https://api.staging.awardsplatform.com';
+
+    private $apiKey;
+
+    public static $emailAlreadyExists = 422;
```

Comment:
This feels a bit out of place. This is a generic client with just get & post type of methods. The only possible 422 is that the email doesn't exist?

Reply:
yep, removed. the 422 error is a bit "generic", we are not responding specific error codes on each error case